### ISSUE TITLE
bs4.FeatureNotFound exception after installation

### ISSUE THREAD

**Issue by kuruoujou** [Utagai/spice#20]
Hello!

It appears that immediately after installation on a fresh python 3 install (or in a fresh virtualenv, as in this case), not all of the requirements are installed via pip:
```
Traceback (most recent call last):
  File "./spice-test.py", line 7, in <module>
    results = spice.search("FLIP FLAPPERS", spice.get_medium('anime'), creds)
  File "/home/kuroshi/spice-test/lib/python3.5/site-packages/spice_api/spice.py", line 165, in search
    query_soup = BeautifulSoup(search_resp.text, 'lxml')
  File "/home/kuroshi/spice-test/lib/python3.5/site-packages/bs4/__init__.py", line 165, in __init__
    % ",".join(features))
bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: lxml. Do you need to install a parser library?
```

Running `pip install lxml` solves the problem of course, but it might be something you want to add to the `install_requires` list in `setup.py`.

**Comment by exxarah** [comment]
This is fixed by #19

**Comment by Utagai** [comment]
Coincidentally, I have just returned to this project to do work on it, I guess it's your lucky day! I've merged #19 so this should be fixed.

Thank you @CyradisNyx.
